File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 10/VB/ObjectDataSourceVB/App_Code/ClassProduct.vb
Imports Microsoft.VisualBasic
Imports System.Data
Imports System.Data.SqlClient

Public Class ClassProduct

    Public Function getinfo() As DataSet

        Dim con As New SqlConnection("Data Source=PUNEET-PC\SQLEXPRESS;Initial Catalog=northwnd;Integrated Security=True")
        Dim da As New SqlDataAdapter("Select * from Products", con)
        Dim ds As New DataSet()
        da.Fill(ds)
        Return ds

    End Function


End Class

File: CS/CS.NET-VB.NET/ASP.NET 4.0 Book/Source/Chapter 10/CS/ObjectDataSource/App_Code/ClassProduct.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Data;
using System.Data.SqlClient;


/// <summary>
/// Summary description for ClassProduct
/// </summary>
public class ClassProduct
{
    public DataSet getinfo()
    {
       
        SqlConnection con = new SqlConnection("Data Source=PUNEET-PC\\SQLEXPRESS;Initial Catalog=northwnd;Integrated Security=True");
        SqlDataAdapter da = new SqlDataAdapter("Select * from Products", con);
        DataSet ds = new DataSet();
        da.Fill(ds);
        return ds;
    }
	
}